cqf-fhir-cr/src/test/java/org/opencds/cqf/fhir/cr/measure/r4/MeasureScorerTest.java

```diff
@@ -84,6 +155,15 @@ MeasureReportGroupComponent group(MeasureReport measureReport, String id) {
                 .get();
     }
 
+    public MeasureReport.MeasureReportGroupStratifierComponent getStratumById(
+            MeasureReport measureReport, String groupId, String stratifierId) {
+        var group = group(measureReport, groupId);
+        return group.getStratifier().stream()
+                .filter(g -> g.getId().equals(stratifierId))
+                .findFirst()
+                .get();
+    }
+
     public List<Measure> getMyMeasures() {
```

Comment:
question: does this work for Measures that have some other basis than "Patient"? Have we validated it works for encounters, as an example? How about cohort Measures?

Reply:
cohort measures should not apply because they don't calculate a measure score (no num or den), it does however add extensions still to the measureScore.group for cohorts. This should be removed

Reply:
encounter measure like CMS111 has a test case and does not currently add the new extensions and does not calculate measureScore

Reply:
CMS111 is continuous variable, so like cohort, doesn’t apply to measureScoring

DischargedonAntithromboticTherapyFHIR is proportion using that same extension for encounter and does appropriately get scored using newly added logic.